container/go/cmd/digester/digester.go

```diff
@@ -21,33 +21,35 @@ import (
 	"log"
 	"os"
 
+	"github.com/bazelbuild/rules_docker/container/go/pkg/compat"
 	"github.com/bazelbuild/rules_docker/container/go/pkg/oci"
 	"github.com/bazelbuild/rules_docker/container/go/pkg/utils"
 )
 
 var (
-	dst        = flag.String("dst", "", "The destination location of the digest file to write to.")
-	imgTarball = flag.String("tarball", "", "Path to the image tarball as generated by docker save. Required if --config was not specified.")
-	imgConfig  = flag.String("config", "", "Path to the image config JSON file. Required if --tarball was not specified.")
-	format     = flag.String("format", "", "The format of the uploaded image (Docker or OCI).")
-	layers     utils.ArrayStringFlags
+	dst          = flag.String("dst", "", "The destination location of the digest file to write to.")
+	imgTarball   = flag.String("tarball", "", "Path to the image tarball as generated by docker save. Required if --config was not specified.")
+	imgConfig    = flag.String("config", "", "Path to the image config JSON file. Required if --tarball was not specified.")
+	baseManifest = flag.String("manifest", "", "Path to the manifest of the base image. This should be the very first image in the chain of images and is only really required for Windows images with a base image that has foreign layers.")
+	format       = flag.String("format", "", "The format of the uploaded image (Docker or OCI).")
+	layers       utils.ArrayStringFlags
 )
 
 func main() {
-	flag.Var(&layers, "layer", "The list of paths to the compressed layers of this docker image, only with --format=Docker.")
+	flag.Var(&layers, "layer", "One or more layers with the following comma separated values (Compressed layer tarball, Uncompressed layer tarball, digest file, diff ID file). e.g., --layer layer1.tar.gz,layer1.tar,<file with digest>,<file with diffID>.")
```

Comment:
should this be layer001.tar.gz,layer001.tar,  ?

Reply:
Removed the number instead as discussed offline